### ISSUE TITLE
implement improvements from ty's first conversation review

### ISSUE THREAD

**Issue by erohmensing** [RasaHQ/rasa-demo#480]
a collection issue for issues 360-396

**Comment by akelad** [comment]
um, should we be linking to them?

**Comment by erohmensing** [comment]
yes, but after closing all of them I couldn't deal with manually copy pasting 36 links yet 😄

**Comment by erohmensing** [comment]
#360
#361
#362
#363
#364
#365
#366
#367
#368
#369
#370
#371
#372
#373
#375
#376
#377
#380
#381
#382
#383
#384
#385
#386
#387
#388
#389
#390
#391
#392
#393
#394
#395
#396
